script/MainnetWrappedStakedObol.sol

```diff
@@ -0,0 +1,26 @@
+// SPDX-License-Identifier: MIT
+pragma solidity 0.8.28;
+
+import {BaseWrappedStakedObolDeploy} from "script/BaseWrappedStakedObol.sol";
+import {IERC20} from "@openzeppelin/contracts/token/ERC20/IERC20.sol";
+
+contract MainnetWrappedStakedObolDeploy is BaseWrappedStakedObolDeploy {
+  function _getWrapperConfig() internal virtual override returns (ObolWrapperParams memory) {
+    uint256 _prefundAmount = 1e18;
+
+    address _expectedWrappedLstAddress =
+      vm.computeCreateAddress(deployer, vm.getNonce(deployer) + 1);
+    vm.broadcast(deployer);
+    IERC20(0x6590cBBCCbE6B83eF3774Ef1904D86A7B02c2fCC).approve(
```

Comment:
This is a bikeshed, and not worth correcting in this case, but for future reference: I think executing a side-effect in a method that starts with "_get" is misleading. This operation seems like it belongs in the base deploy script anyway. Again, not worth changing at this point, but just a nit for future reference.